keras_cv/backend/ops.py

```diff
@@ -16,8 +16,46 @@
 if multi_backend():
     from keras_core.operations import *  # noqa: F403, F401
 else:
+    import tensorflow as tf
     from keras_core.backend.tensorflow import *  # noqa: F403, F401
     from keras_core.backend.tensorflow.core import *  # noqa: F403, F401
     from keras_core.backend.tensorflow.math import *  # noqa: F403, F401
     from keras_core.backend.tensorflow.nn import *  # noqa: F403, F401
     from keras_core.backend.tensorflow.numpy import *  # noqa: F403, F401
+
+    def split(x, indices_or_sections, axis=0):
+        return tf.split(x, indices_or_sections, axis=axis)
+
+    def max(x, axis=None, keepdims=False, initial=None):
+        # The TensorFlow numpy API implementation doesn't support `initial`
+        # so we handle it manually here.
+        if initial is not None:
+            return tf.math.maximum(
+                tf.reduce_max(x, axis=axis, keepdims=keepdims), initial
+            )
+
+        # TensorFlow returns -inf by default for an empty list, but for
+        # consistency with other backends and the numpy API we want to
+        # throw in this case.
+        size_x = tf.size(x)
+        tf.assert_greater(
+            size_x,
+            tf.constant(0, dtype=size_x.dtype),
+            message="Cannot compute the max of an empty tensor.",
+        )
+
+        return tf.reduce_max(x, axis=axis, keepdims=keepdims)
+
+    def concatenate(xs, axis=0):
```

Comment:
For this (and same for a few below), we can `from tensorflow import concat as concatenate` (same for shape, range).

Reply:
I rebased on top of your core branch and pulled the changes which do this. Let me know if I missed anything here!